addOns/pscanrules/CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - The "Viewstate without MAC Signature (Unsure)" alert will now only be raised at Low Alert Threshold (Issue 7230).
 - The Content Security Policy scan rule will now alert when "unsafe-hashes" are allowed.
 
+### Fixed
+-  I meant having Correct parameter and evidence for Cookie without SameSite Attribute when SAMESITE was set to None (Issue 7358).
```

Comment:
Oops, bad copy/paste

Reply:
Drop the first three words

Reply:
yeah

Reply:
No worries, I’ve done that exact thing too